bundles/io/org.openhab.io.rest/src/main/java/org/openhab/io/rest/internal/listeners/ResourceStateChangeListener.java

```diff
@@ -54,6 +54,7 @@
 	
 	static ScheduledFuture<?> executorFuture;
 	
+	protected Item lastChange;
```

Comment:
OK sounds good, since I don't really have a good way to test this without writing a WS client, I will trust you have thoroughly tested that both the websocket functionality and the existing rest functionality work correctly.  Thanks for your patience, its been a busy month.